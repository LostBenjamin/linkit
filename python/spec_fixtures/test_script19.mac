
(((testing 5 `X+1` `X+2`)))

[[testing X Y Z
  [testingX]
    X:red
    Y:grn
    Z:blu
    flu
]]


