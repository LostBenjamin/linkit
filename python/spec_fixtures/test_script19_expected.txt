4:set:0:red:1:grn:2:blu:flu
5:set:1:red:2:grn:3:blu:flu
6:set:2:red:3:grn:4:blu:flu
7:set:3:red:4:grn:5:blu:flu
8:set:4:red:5:grn:6:blu:flu

